ES/ESath007.xml

```diff
@@ -252,77 +418,84 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                              Hāymānot), namely: priests Tomas, ʿAmda Mikāʾel, Tasfā ʾIyasus, ʿAṣfa
+                              Krǝstos, Gabra Krǝstos, Rǝʾsa Hāymānot, Takla Hāymānot. To note that,
                               here, the verb used to indicate the acquisition of the Ms. is አጥረየ
-                              (ʾaṭräyä). In Additio 1, it is ተሳየጠ (täsayäṭä). </note>
+                              (ʾaṭraya). In Additio 1, it is ተሳየጠ (tasāyaṭa). </note>
                         </item>
                         <item xml:id="a3">
-                           <locus target="#139ra">f. 139ra</locus>
-                           <desc>(below the text): Supplication note (?).</desc>
-                           <quote xml:lang="gez"/>
-                           <quote xml:lang="en"/>
+                           <locus target="#139ra">f. 139ra (below the text)</locus>
+                           <desc type="Supplication">Supplication note (?).</desc>
+                           <q xml:lang="gez"/>
+                           
                            <note> The name(s) to whom the supplication is dedicated are not readable
                               because erased. </note>
                         </item>
                         <item xml:id="a4">
-                           <locus target="#139rb">f. 139rb</locus>
-                           <desc>-rc: salām-hymn for St Mary (cp. ChRep 191, n</desc>
-                           <locus target="#14">14</locus>), incomplete. <quote xml:lang="gez">
-                              <locus target="#139rb">139rb-</locus>rc: ሰላም፡ ለኪ፡ ማርያም፡ ስጣሕት፡
-                              ቀስተተ፡[sic] መሐላሁ፡ ለኖኅ፡…</quote>
-                           <quote xml:lang="en"/> Cp. the same text in <locus from="118rb" to="119rb">118rb-119rb</locus>
-                           <note> of Ms. MKL-001 (Text II-2 in “Maṣḥafa gǝbra ḥǝmāmāt”). The note
+                           <locus target="#139r">f. 139rb-rc</locus>
+                           <title ref="LIT2730RepCh14" type="incomplete"></title>
+                           <desc type="GuestText"> salām-hymn for St Mary (cp. ChRep 191, n 14), incomplete.</desc>
+                           <q xml:lang="gez">
+                              ሰላም፡ ለኪ፡ ማርያም፡ ስጣሕት፡
+                              ቀስተተ፡[sic] መሐላሁ፡ ለኖኅ፡…</q>
+                            Cp. the same text in <locus from="118rb" to="119rb">118rb-119rb</locus>
+                           <note>See also the same text in <ref type="mss" corresp="ESmkl001#ms_i1.2.2"></ref></note>
+                           <note>The note
                               contains rubrications. </note>
                         </item>
-                        <item xml:id="e1">- In some supplication formulas, the names have been
+                        <item xml:id="e1"><desc type="Supplication">- In some supplication formulas, the names have been
```

Comment:
I'm not sure whether the type Supplication should be given here - the item is on the erasure, not on the supplication